### PR TITLE
Updated Authentication page

### PR DESCRIPTION
**Pull request by Anurag021**
Added Authentication page in Ecommerce website, added login and signup page.
--added files
- login.css
- login.html
- signup.html

### Preview Link
https://focused-stonebraker-4f858f.netlify.app/authentication/signup
https://focused-stonebraker-4f858f.netlify.app/authentication/login

### CODE REVIEW

File: Components/login.css [hunk 1]
```diff
@@ -1,14 +1,15 @@
 .login-container{
     background-color: var(--color-primary);
     width: auto;
```
Comment (MayankKumar10) [inline Components/login.css:3]: width is by default Auto try to avoid using that

File: Components/login.css [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
-    height: 35rem;
+    height: 30rem;
 }
 .input-form{
-    width: 30rem;
+    width: 27rem;
     border-radius: 1rem;
     height: 30rem;
     align-self: center;
     margin-top: 2rem;
```
Comment (MayankKumar10) [inline Components/login.css]: use shorthand property to define margin, padding i.e Top, Right, Bottom, Left

**Review by MayankKumar10 — APPROVED** [review]
Everything else Looks Good

**Review by nayyyhaa — APPROVED** [review]
One suggestion in UI, you could give more white space between the content & the footer. The code looks good & readable.  Nice work!

File: Components/login.css [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    background-image: url(/Assests/login-bg.png);
+}
+.input-form label, input{
+    font-size: large;
+}
+.input-form .checkbox-input-form{
+    width: auto;
+}
+.login-btn, .sign-up-btn{
+    font-weight: bold;
```
Comment (Developers-codz) [inline Components/login.css]: use font-weight value instead of bold bolder

File: app.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+var passwordEyeBtn = document.querySelector('.password-check')
+var passwordForm = document.querySelector('.input-password')
```
Comment (Developers-codz) [inline app.js]: using id selector will be better for js

**Review by Developers-codz — APPROVED** [review]

**Review by Sahil25061999 — APPROVED** [review]
same  changes  as suggested above.
Just a suggestion for UI the input width can be increased or the form width can be decreased.

File: Components/login.css (hunk 1 above)
Comment (Anurag021) [reply Components/login.css:3]: width is inherited here, I am overwriting that value with 0; but thanks for pointing it out; i was not aware of width default value...😁

File: Components/login.css (hunk 2 above)
Comment (Anurag021) [reply Components/login.css]: Thanks, done.

File: Components/login.css (hunk 3 above)
Comment (Anurag021) [reply Components/login.css]: Done, Thanks.

File: app.js (hunk 4 above)
Comment (Anurag021) [reply app.js]: Done, Thanks..

**Pull request merged**